scripts/contactlist.py

```diff
@@ -14,25 +14,15 @@
 
 active_pcs = Pc.objects.exclude(type__slug='admin').filter(is_active=True)
 
-
-def clean_email(email):
-    """ Remove None and make all lowercase"""
-    if email is None:
-        return ''
-    return email.lower()
-
-
 contacts = set()
 for pc in active_pcs:
     sname = pc.station.name
     sn = str(pc.station.number)
     contact_info = pc.station.contactinformation
-    email1, email2 = map(clean_email, (contact_info.email_work, contact_info.email_private))
-
-    email1 = email1.lower()
-    email2 = email2.lower()
+    email1 = contact_info.email_work.lower() if contact_info.email_work else ''
```

Comment:
Thanks! I like your changes!

(I'm guessing this will fail without
`from __future__ import print_function` but we'll manage)

Reply:
Nope, in recent python 2.7.x `print()` works fine without that.